### ISSUE TITLE
worker-windows job marked as "failing" when atc is not up

### ISSUE THREAD

**Issue by flavorjones** [pivotal-cf-experimental/concourse-windows-release#8]
Given a bosh-deployed concourse cluster running 3.9.2 for all components,
When I `bosh stop web` to stop the atc process,
The windows workers are marked by bosh as "failing" or "stopped",
But the linux workers are still deemed "running"

---

(In this case, I stopped the web job because I needed to renew my letsencrypt SSL cert.)

Here's what it looks like immediately after I `bosh stop web`:

```
$ bosh vms
Using environment 'https://35.190.159.215:25555' as client 'admin'

Task 3462. Done

Deployment 'concourse'

Instance                                             Process State  AZ  IPs         VM CID                                   VM Type   Active  
db/11eb57d4-89f4-4745-9439-7a310aa25628              running        z1  10.0.16.5   vm-dd0751ea-0f60-451a-7491-50a7670afa20  g1-small  false  
web/15659ac3-a321-40c6-b98a-5e25ec52113a             stopped        z1  10.0.16.4   vm-a90c2715-e66d-4981-73e3-6bb158c4c212  f1-micro  false  
worker-windows/1e56f81a-dfbd-4013-bd02-a746fe54107a  failing        z1  10.0.16.12  vm-5410171a-f7e7-4d0c-4cc5-80bdf5ac5fa0  g1-small  false  
worker-windows/d62bb469-d4c2-4920-88ed-3644ce87dd23  stopped        z1  10.0.16.8   vm-585bb496-b16a-4328-45b2-59ea980e46c7  g1-small  false  
worker/20608120-d78b-442a-a224-fda545f798c2          running        z1  10.0.16.13  vm-41935e7f-ff83-4a8b-4eeb-aca3528e0274  g1-small  false  
worker/53adc981-ca4c-4c56-be96-4d66375f05f4          running        z1  10.0.16.14  vm-dd14f83f-6e49-4986-4ba3-c827d3cfa9b9  g1-small  false  
worker/94debc2a-771f-4f0e-9f4d-5bc9829ef47d          running        z1  10.0.16.15  vm-3a57d8dd-140f-4aac-68bd-18d57b17f655  g1-small  false  
worker/9ae7b63e-2c20-4fba-bba3-ad1525765669          running        z1  10.0.16.16  vm-c3fd174f-cf36-4c67-463f-a82a03bec2a1  g1-small  false  
worker/bb58f122-b196-443c-9454-d7fe6652f5d8          running        z1  10.0.16.17  vm-2a475b0c-c310-4984-44b5-06bcbee795c1  g1-small  false  
worker/c058c641-503d-4b39-89ec-e7de82f20219          running        z1  10.0.16.10  vm-b91aacfd-0060-4c37-4da4-1a04c8057d5d  g1-small  false  
worker/c26c88ce-b031-4c1d-b1de-6084907840c3          running        z1  10.0.16.11  vm-990777dd-d51f-47b5-6318-c48b1f868f99  g1-small  false  
worker/c3416081-2d46-47a1-bdf9-81d68fb70932          running        z1  10.0.16.6   vm-d599af74-9ed7-49c4-51dc-d8cfe1839606  g1-small  false  
worker/c635cffc-fae8-472e-84b3-969ce5df247d          running        z1  10.0.16.7   vm-37436e81-5f15-4659-5304-3806080384f8  g1-small  false  
worker/fd1f081b-1bc8-4d24-b58f-37660137d8d5          running        z1  10.0.16.9   vm-e95dfb48-acc0-45b9-483c-57545b0899b8  g1-small  false  
```

I reproduced this condition a couple of times. I'm attaching log files from worker-windows/0 in case it's helpful.

[worker-windows.0.zip](https://github.com/pivotal-cf-experimental/concourse-windows-worker-release/files/1864524/worker-windows.0.zip)

**Comment by sunjayBhatia** [comment]
@flavorjones We believe this is fixed in the `3.10.0` release as it ensures that connections are automatically retried (see: https://github.com/concourse/worker/commit/265f42d87a25cbc2de444285779c34c0ec05d5d0)

We are upgrading our concourse to `3.10.0` to confirm

**Comment by sunjayBhatia** [comment]
@flavorjones upgrading to `3.10.0` fixes this for us, let us know if you have any other issues

Closing this until then
